File: app/src/main/java/com/zxcx/zhizhe/ui/search/result/SearchResultContract.java
package com.zxcx.zhizhe.ui.search.result;

import com.zxcx.zhizhe.mvpBase.IGetPresenter;
import com.zxcx.zhizhe.mvpBase.GetView;

import java.util.List;

public interface SearchResultContract {

    interface View extends GetView<List<SearchCardBean>> {
        void searchCardBagSuccess(List<SearchCardBagBean> list);
    }

    interface Presenter extends IGetPresenter<List<SearchCardBean>> {
        void searchCardBagSuccess(List<SearchCardBagBean> list);
    }
}

File: app/src/main/java/com/zxcx/zhizhe/ui/search/result/SearchResultContract.kt
package com.zxcx.zhizhe.ui.search.result

import com.zxcx.zhizhe.mvpBase.GetView
import com.zxcx.zhizhe.mvpBase.IGetPresenter

interface SearchResultContract {

    interface View : GetView<MutableList<SearchResultBean>>

    interface Presenter : IGetPresenter<MutableList<SearchResultBean>>
}

